packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -269,39 +266,35 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
             collateralAvailable,
           );
 
-          const { curAuctionPrice, currencySeat, remainingProceedsGoal } =
+          const { curAuctionPrice, bidSeat, remainingProceedsGoal } =
```

Comment:
I suppose this change to the durable storage schema is another one to keep in mind, along with the network protocol.